packages/omg-js-rootchain/src/txUtils.js

```diff
@@ -85,7 +96,7 @@ async function setGas (web3, txDetails) {
         txDetails.gas = await web3.eth.estimateGas(txDetails)
       }
     } catch (err) {
-      throw new Error(`Error estimating gas: ${err}`)
+      console.warn(`Error estimating gas: ${err}`)
```

Comment:
i remember @kevin saying we should throw here, because the transaction will definitely fail so shouldnt swallow

Reply:
if we always throw, the real revert reason will never pop up because of web3 swallow it and it is super hard to debug what is happening. Also, estimation can be wrong sometimes like processExits.

Reply:
Ah yes good point. Okay 👌